src/build/intro-currency-runtime.js

```diff
@@ -0,0 +1,43 @@
+/**
```

Comment:
Can you have a single template for all the intro file headers and generate them at build time? There's a lot of duplication here.